GCC 4.2.1 doesn't understand "-Wpedantic".

Move examples to the right place.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -389,7 +389,7 @@ include(CompilerFlags)
 if(NOT UA_COMPILE_AS_CXX AND (CMAKE_COMPILER_IS_GNUCC OR "x${CMAKE_C_COMPILER_ID}" STREQUAL "xClang"))
     # Compiler
     add_definitions(-std=c99 -pipe
-                    -Wall -Wextra -Wpedantic
+                    -Wall -Wextra
                     -Wno-static-in-inline # clang doesn't like the use of static inline methods inside static inline methods
                     -Wno-overlength-strings # may happen in the nodeset compiler when complex values are directly encoded
                     -Wno-unused-parameter # some methods may require unused arguments to cast to a method pointer
@@ -1239,7 +1239,7 @@ if (UA_PACK_DEBIAN)    
         "examples/nodeset"
         "examples/pubsub")
 
-    set(open62541_install_examples_dir share/open62541/examples)
+    set(open62541_install_examples_dir share/examples/open62541)
     install(DIRECTORY ${UA_install_examples_dirs} COMPONENT examples DESTINATION ${open62541_install_examples_dir} USE_SOURCE_PERMISSIONS)
 endif()
 
